Nautono/RTOS_framework_v1/src/sail_tasksinit.h

```diff
@@ -37,6 +36,7 @@ extern unsigned char watchdog_reset_value;
 
 #define GPS_PRIORITY					  tskIDLE_PRIORITY + 3
 #define WEATHER_SENSOR_PRIORITY           tskIDLE_PRIORITY + 4
+#define WIND_PRIORITY					  tskIDLE_PRIORITY + 4
```

Comment:
looks like there's some white space funny business going on here. Are you a tab or 4 spaces guy?